hr_employee_document/README.rst

```diff
@@ -25,13 +25,21 @@ HR Employee Document
 
 |badge1| |badge2| |badge3| |badge4| |badge5| 
 
-This module allows to attach documents to the employee profile.
+This module displays a button on the employee's profile to view linked attachments.
```

Comment:
```suggestion
This module displays a button on the employee's profile to view linked attachments, both
for HR officers/managers and the own employee, so you can use it as employee document storage.
```

Add in usage that you can attach the document as HR office/manager or click on the smart-button and create a new attachment.